pages/Documents/ConversationalAI/BotAnalytics/macs.md

```diff
@@ -61,49 +61,49 @@ There are several charts on the MACS page to help you understand the bot’s per
 
 The **Overall trend** chart displays the average MACS for the bot. Use this graph to understand the bot’s overall MACS trend for the specified date range.
 
-<img style="width:700px" src="img/ConvoBuilder/macs_chart_overalltrend.png" alt="">
+<img style="width:700px" src="img/ConvoBuilder/macs_chart_overalltrend.png" alt="The Bot conversation quality chart displaying the overall  trend">
 
 #### Trend detail
 
 The **Trend detail** chart displays the distribution for each MACS scale: 1, 2 and 3. Use this chart to understand the trend of each MACS scale for the specified date range.
 
-<img style="width:700px" src="img/ConvoBuilder/macs_chart_trenddetail.png" alt="">
+<img style="width:700px" src="img/ConvoBuilder/macs_chart_trenddetail.png" alt="The Bot conversation quality chart displaying the trend detail">
 
 #### Interactions to review
 
 The **Interactions to review** chart displays the top eight, most used interactions that appeared in low-scoring conversations with issues. Use this to determine if bot interaction tuning might be needed. An interaction displayed in this graph might not be the cause of issues, but the relationship can suggest that further investigation is needed.
 
-<img style="width:500px" src="img/ConvoBuilder/macs_chart_interactionstoreview.png" alt="">
+<img style="width:500px" src="img/ConvoBuilder/macs_chart_interactionstoreview.png" alt="The Interactions to review chart">
 
 #### Issue types
 
 The **Issue types** chart displays the distribution of issues types that were found in conversations within the specified date range. The issue types are potential friction points (failure points) detected in conversations; they affect the conversation’s MACS.
 
-Use these issues types to understand how the consumer’s experience was impeded and to discover bot tuning opportunities. For possible fixes, see [here](https://knowledge.liveperson.com/data-reporting-meaningful-automated-conversation-score-(macs).html#how-is-macs-calculated).
+Use these issues types to understand how the consumer’s experience was impeded and to discover bot tuning opportunities. For possible fixes, see [this topic](https://knowledge.liveperson.com/data-reporting-meaningful-automated-conversation-score-(macs).html#how-is-macs-calculated) in the Knowledge Center.
```

Comment:
(Optional: It seems we could also write, “…learn [how MACS is being calculated]().”)